packages/react-core/src/components/Select/Select.tsx

```diff
@@ -164,8 +166,76 @@ const SelectBase: React.FunctionComponent<SelectProps & OUIAProps> = ({
   );
 };
 
-export const Select = React.forwardRef((props: SelectProps, ref: React.Ref<any>) => (
-  <SelectBase innerRef={ref} {...props} />
-));
+SelectBase.displayName = 'SelectBase';
+
+/**
+ * Extends SelectBaseProps
+ */
+
+export interface SelectProps extends Omit<SelectBaseProps, 'toggle'> {
+  /** Renderer for a custom select toggle. Forwards a ref to the toggle. */
+  toggle?:
+    | SelectToggleProps
+    | ((toggleRef: React.RefObject<any>, selected: any, isOpen: boolean, toggleOpen: () => void) => React.ReactNode);
```

Comment:
This second toggle prop definition confused me a little at first. I know its a way to customize the simplified toggle without also requiring the open state and click handler, so maybe we can update the description here a little to indicate that (or to indicate that full control of the toggle and handlers should refer to SelectBase).

WYDT about renaming `toggleOpen` to `onToggleHandler`?

Reply:
I think `onToggleHandler` is a good suggestion, I'll change that.